packages/aragon-wrapper/src/core/aragonOS/index.js

```diff
@@ -17,6 +17,12 @@ const APP_MAPPINGS = {
   [oldWrongAppId('evmreg')]: 'EVM Script Registry'
 }
 
+const KERNEL_NAMESPACES = {
```

Comment:
Would you consider using a Map here:
```js
const KERNEL_NAMESPACES = new Map([
  [soliditySha3('core'), 'Core'],
  [soliditySha3('app'), 'Default apps'],
  [soliditySha3('base'), 'App code'],
])

function getKernelNamespace (hash) {
  if (KERNEL_NAMESPACES.has(hash)) {
    return { name: KERNEL_NAMESPACES.get(hash), hash }
  }
}
```

Reply:
👍 we could update `APP_MAPPINGS` to be a map too.

Reply:
Updated in 503a1c8 , I will merge this as I have some tests on another branch `postprocessRadspecDescription`